tests/src/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -1242,4 +1242,43 @@ class WskBasicUsageTests
             }
             tmpProps.delete()
     }
+
+    behavior of "Wsk cli list"
+
+    it should "wsk cli list entities by name alphabetically by default" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+
+            val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+            try {
+                val namespace = wsk.namespace.list().stdout.trim.split("\n").last
+                val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+                wsk.cli(Seq("property", "set", "-i", "--apihost", wskprops.apihost, "--auth", wskprops.authKey,
```

Comment:
using a config file seems like overkill - this is to avoid adding `sort` to the `Wsk.list` method?